packages/front-end/services/UserContext.tsx

```diff
@@ -268,7 +268,7 @@ export function UserContextProvider({ children }: { children: ReactNode }) {
     growthbook.setAttributes({
       id: data?.userId || "",
       name: data?.userName || "",
-      admin: data?.admin || false,
+      superAdmin: data?.superAdmin || false,
```

Comment:
Do we want to rename the key here?

Reply:
yeah, renaming is fine. We might need to make some changes to our feature flag rules, but there aren't many